micromasters/utils.py

```diff
@@ -118,6 +118,16 @@ def first_matching_item(iterable, predicate):
     return next(filter(predicate, iterable), None)
 
 
+def remove_falsey_values(iterable):
```

Comment:
It's just a clearly explained shorthand. `filter(None, iterable)` does the same thing, but I would argue it's not a commonly known trick that `filter` can accept `None` instead of a function as its first arg. `remove_falsey_values` is pretty explicit